README.md

```diff
@@ -48,18 +63,20 @@ end
 
 ## Usage
 
-### Cache whatever you want by `cache_at` method
+### The `cache_at` method
+
+Use the `cache_at` method to cache whatever you want. Specify a cache on the model:
 
 `cache_at(name, query = nil, options = {})`
 
-Specify a cache on the model.
- - name: the attribute name.
- - query: how to get data on cache miss. It will be set automatically if the name match an association or an attribute.
+Parameters:
+ - name: the attribute name
+ - query: how to get data on cache miss. It will be set automatically if the name matches an association or an attribute.
  - options: see [here](#options)
 
-### Asscess the cached attributes
+### Access the cached attributes
 
-The `cacher` is defined as `class method` and `instance method` of Model. You could call the method and get the cacher instance, e.g. `User.cacher` or `user.cacher`. An attribute will define a method on cacher, cached attributes are asscessable via it, e.g. `user.cacher.the_attribute_name`.
+The `cacher` is defined as a `class method` and `instance method` of the model. You can call the method and get the cacher instance (e.g. `User.cacher` or `user.cacher`). Cached attributes can be accessed by defining a method on cacher (e.g. `user.cacher.the_attribute_name`).
```

Comment:
Cached attributes can be accessed by the method defined by `cache_at`, not by defining a method on cacher.

You could see the [example](#Basic-Example). This line (`cache_at :something_you_want_to_cache, ->{ xxx }`) will define `something_you_want_to_cache` method on cacher.

Reply:
That makes sense. Would you say in the sentence "cached attributes are accessible via it", "it" is referring to "user.cacher"?  I just wanted to replace "it" with what it's actually referring to so it can be straightforward.

Reply:
I would choose `cached attributes are accessible via it`. But I just feel that this section is unstructured and may need to be rewrite. Do you think the following is more understandable:

Cached attributes can be accessed by the method defined by `cache_at`. And all methods will be defined on `Cacher` instead of `ActiveModel` to avoid method name collision. You can access the `cacher` from class or from instance (e.g. `User.cacher` or `user.cacher`), then access the method defined by `cache_at` (e.g. `user.cacher.the_attribute_name`).

Reply:
I realized the first sentence and the last sentence are both referring to the same idea that "cached attributes can be accessed by cache_at" so can we remove one of the sentences?
Something like:
"To avoid method name collision, all methods will be defined on the Cacher instead of ActiveModel. You can access the cacher from the class or from the instance (e.g. User.cacher or user.cacher), then access cached attributes via the method defined by cache_at (e.g. user.cacher.the_attribute_name)."

Reply:
I think `get` will be more suitable than `call` if you don't want to use the same word `access` twice.